lib/World.lua

```diff
@@ -7,6 +7,8 @@ local assertValidComponent = Component.assertValidComponent
 local archetypeOf = archetypeModule.archetypeOf
 local areArchetypesCompatible = archetypeModule.areArchetypesCompatible
 
+local QueryResult = require(script.Parent.query)
```

Comment:
Any reason why the file was named "query" instead of "QueryResult"?

Reply:
Mostly because I am conflicted on what to name. I would much prefer calling the file query, but that might mean we have to rename the struct to query as well which I am not sure how I feel about that yet. 

What do you suggest?